### ISSUE TITLE
Display Icon/Image inside bubble.

### ISSUE THREAD

**Issue by yashkumarsharma** [kauffecup/react-bubble-chart#4]
Can we extend the functionality by providing an option to display icon/image inside the bubble?

**Comment by kauffecup** [comment]
absolutely! i unfortunately am stretched a bit thin right now but will leave this issue open as a reminder for future-me!

also, if you'd like to get started on it in the meantime i can absolutely help!

**Comment by vitaly87** [comment]
@kauffecup  can you please help with adding svg / images on each node?

**Comment by yashkumarsharma** [comment]
@vitaly87 - 
Can you check if this is helpful - https://github.com/yashkumarsharma/react-bubble-chart/tree/0.3.0
npm - https://www.npmjs.com/package/react-bubble-chart-wi

If Yes and it is working fine for you, I will raise a PR in the original repository for @kauffecup.

**Comment by vitaly87** [comment]
@yashkumarsharma  I didnt get you , what need to be done in order to set icons for each bauble

**Comment by yashkumarsharma** [comment]
Steps: (I have not tested it at my end)
a) `npm install --save react-bubble-chart-wi`
b) The place where you are providing text for bubble, will now accept html as well.

For a better understanding review 
- https://github.com/kauffecup/react-bubble-chart/compare/master...yashkumarsharma:0.3.0
File -src/ReactBubbleChartD3.js

Regards,
Yash
